contracts/src/chainlink/solidity/mocks/TestERC20.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: Apache-2.0.
+pragma solidity ^0.6.12;
+
+import "../../../../vendor/starkware-libs/starkgate-contracts/src/starkware/solidity/tokens/ERC20/ERC20.sol";
+
+/*
+  An ERC20 for testing where anyone can set the balance for everyone.
+*/
+contract TestERC20 is ERC20 {
+  function setBalance(address account, uint256 amount) external {
```

Comment:
Not sure if this custom ERC20 is required. What's the purpose of adding this method?

I must be missing something as currently I see you added the method, and you are just testing this specific method.